clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperations.groovy

```diff
@@ -57,4 +57,8 @@ final class AtomicOperations {
   public static final String RUN_JOB = "runJob"
   public static final String DESTROY_JOB = "destroyJob"
   public static final String CLONE_JOB = "cloneJob"
+
+  // Serialize operations
+  public static final String SERIALIZE_APPLICATION = "serializeApplication"
+  public static final String DESERIALIZE_APPLICATION = "deserializeApplication"
```

Comment:
Would it make sense to support operating on Spinnaker 'projects' as well? They allow the user to collect Applications/Clusters/Pipelines, and that may make for a nice grouping mechanism to use when determining scope.

We don't need to answer this question necessarily while working on this PR, just wanted to mention it for now.